backend-models/service-tests/courseedservice-tests.txt

```diff
@@ -0,0 +1,258 @@
+
+courseedservice.php
+
+---------------------------------------------------------------------------------------------------------------------
+- Insert into course (line 79)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $opt = "NEW";
+    $userid = "UNK"; or $userid = !NULL;
```

Comment:
you need to have real values for each variable, it has to be something that can be entered by the person writing the test program. That person should not have to figure out what value CAN be entered, that is the whole point of this document.

Reply:
this of course does not include the last variable, i know that in that case no values are presented at the time of writing this